src/main/generic/Core.js

```diff
@@ -1,9 +1,9 @@
 class Core {
-    constructor() {
-        return this._init();
+    constructor(options) {
+        return this._init(options);
     }
 
-    async _init() {
+    async _init({ walletSeed }) {
```

Comment:
To be honest, I never saw this kind of parameter list.
Do you have a reference for this JS magic?